antarest/study/storage/rawstudy/model/filesystem/matrix/input_series_matrix.py

```diff
@@ -131,3 +132,37 @@ def copy_file(self, target: str) -> None:
         target_path = self.config.path.parent.joinpath(f"{target}{''.join(self._infer_path().suffixes)}")
         target_path.unlink(missing_ok=True)
         shutil.copy(self._infer_path(), target_path)
+
+    def get_file_content(self) -> t.Tuple[bytes, str, str]:
+        """
+        Get file content
+
+        Returns:
+            file content, as bytes
+            file suffix
+            file name
+        """
+        suffix = self.config.path.suffix
+        filename = self.config.path.name
+        if self.config.archive_path:
+            return (
+                read_original_file_in_archive(
+                    self.config.archive_path, self.get_relative_path_inside_archive(self.config.archive_path)
+                ),
+                suffix,
+                filename,
+            )
+        elif self.get_link_path().is_file():
+            target_path = self.config.path.with_suffix(".txt")
+            buffer = io.StringIO()
+            self.parse(return_dataframe=True).to_csv(  # type: ignore
```

Comment:
Note that we have a `dump_dataframe` in `matrix.py` that was created to avoid code duplication where we need to re-use the same arguments of `to_csv` everywhere.
I think we should re-use this method (probably adapting it a bit to allow for an IO object).